pkg/controller/pravega/pravega_controller.go

```diff
@@ -97,12 +97,13 @@ func makeControllerPodSpec(p *api.PravegaCluster) *corev1.PodSpec {
 				ReadinessProbe: &corev1.Probe{
 					Handler: corev1.Handler{
 						Exec: &corev1.ExecAction{
-							Command: util.HealthcheckCommand(9090),
+							Command: util.ControllerReadinessCheck(10080),
 						},
 					},
 					// Controller pods start fast. We give it up to 1 minute to become ready.
-					PeriodSeconds:    5,
-					FailureThreshold: 12,
+					InitialDelaySeconds: 20,
+					TimeoutSeconds:      10,
```

Comment:
timeout of 10 secs for curl is very low... should be about 60 secs...

Reply:
Done.